dashboard/fixtures/current_grades.json

```diff
@@ -0,0 +1,15 @@
+[
+	{
+		"course_key": "course-v1:edX+DemoX+Demo_Course",
+		"passed": true, "percent": 0.97,
+		"letter_grade": "Pass",
```

Comment:
to answer anyway your first question: no, they are not, the letter grade is actually defined in the course grading policy (by the course creators) and could be basically anything.